src/Test.FunctionalTests.Sarif/TestData/Multitool/ValidateCommand/ExpectedOutputs/SARIF2025.ProvideRuleFriendlyName_Invalid.sarif

```diff
@@ -0,0 +1,103 @@
+{
+  "$schema": "https://schemastore.azurewebsites.net/schemas/json/sarif-2.1.0-rtm.5.json",
+  "version": "2.1.0",
+  "runs": [
+    {
+      "tool": {
+        "driver": {
+          "name": "SARIF Functional Testing",
+          "rules": [
+            {
+              "id": "SARIF2025",
+              "name": "ProvideRuleFriendlyName",
+              "shortDescription": {
+                "text": "Each analysis rule should provide a \"friendly name\" in its 'name' property, in addition to the stable, opaque identifier in its 'id' property."
+              },
+              "fullDescription": {
+                "text": "Each analysis rule should provide a \"friendly name\" in its 'name' property, in addition to the stable, opaque identifier in its 'id' property. This helps users see at a glance the purpose of the analysis rule. For uniformity of experience across all tools that produce SARIF, the friendly name should be a single Pascal identifier, for example, 'ProvideRuleFriendlyName'."
+              },
+              "messageStrings": {
+                "None_FriendlyNameMissing": {
+                  "text": "{0}: This rule does not provide a \"friendly name\" in its 'name' property. The friendly name should be a single Pascal identifier, for example, 'ProvideRuleFriendlyName', that helps users see at a glance the purpose of the analysis rule."
+                },
+                "None_FriendlyNameNotAPascalIdentifier": {
+                  "text": "{0}: '{1}' is not a Pascal identifier. For uniformity ofexperience across all tools that produce SARIF, the friendly name should be a single Pascal identifier, for example, 'ProvideRuleFriendlyName'."
+                }
+              },
+              "defaultConfiguration": {
+                "level": "note"
+              },
+              "helpUri": "http://docs.oasis-open.org/sarif/sarif/v2.1.0/sarif-v2.1.0.html"
+            }
+          ]
+        }
+      },
+      "invocations": [
+        {
+          "executionSuccessful": true
+        }
+      ],
+      "artifacts": [
+        {
+          "location": {
+            "uri": "FunctionalTestOutput.ValidateCommand/Inputs.SARIF2025.ProvideRuleFriendlyName_Invalid.sarif",
+            "uriBaseId": "TEST_DIR"
+          }
+        }
+      ],
+      "results": [
+        {
+          "ruleId": "SARIF2025",
+          "ruleIndex": 0,
+          "level": "note",
+          "message": {
+            "id": "None_FriendlyNameMissing",
+            "arguments": [
+              "runs[0].tool.driver.rules[0]",
+              "TEST0001"
```

Comment:
>TEST0001 [](start = 15, length = 8)

This is a good idea, to include the rule id in the message. But it means I need to revise the message text to include this parameter.